provision/provision.sh

```diff
@@ -526,10 +529,8 @@ phpfpm_setup() {
 
 go_setup() {
   if [[ ! -e /usr/local/bin/mailhog ]]; then
-    echo " * Installing GoLang 1.10.3"
-    curl -sO https://dl.google.com/go/go1.10.3.linux-amd64.tar.gz
-    tar -xvf go1.10.3.linux-amd64.tar.gz
-    rm go1.10.3.linux-amd64.tar.gz
+    echo " * Installing GoLang 1.11"
+    curl -so- https://dl.google.com/go/go1.11.linux-amd64.tar.gz | tar zxvf -
```

Comment:
wondering if we can do the same like for nvm also for go. In that way use github to get always the latest version so we don't have to bump that.

Reply:
I think it should be possible, but there’s one thing about Vagrant I haven’t quite got my head around yet: can the architecture vary? If not, I can always download `go${LATEST_RELEASE}.linux-amd64.tar.gz`.

Reply:
the architecture is defined on vagrantfile and I don't think that will change :-)

Reply:
Okay, great. Then I’ll update this so it automatically downloads the latest version.

Reply:
Does `curl --silent "https://api.github.com/repos/golang/go/releases/latest"` give you a 404? I can get the contributors, but not the releases, it seems.

Reply:
As long as it matches the architecture of the VM OS it should be fine, but I'm unaware of any non-`amd64` scenarios, and it would require VVV running on a non-intel/AMD processor

Reply:
Gotcha, thanks.

According to GitHub support, the reason why the release list won’t work for Go is that there are no official releases, just tags. I can’t get the latest tag either, though, so I’m waiting to hear from them.

Reply:
ok so I will waiting before an official test of this pr for this change :-)